app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PasswordResetsController < ApplicationController
+  def create
+    user = User.find_by_token(params[:token])
+    if user
+      user.password = params[:password]
+      user.generate_token
```

Comment:
I would say something like `#remove_token` and you would just `.update_column(:token, nil)`